### PR TITLE
GL-1718.  Update for lower call rates due to zeroed out SNPs

### PR DESCRIPTION
**Pull request by gbggrant**
GL-1718.  This PR updates the Arrays related WDLs to the latest version of picard, which has several small fixes.
1) We have changed the calculation/determination of gtc_call_rate slightly because the way we did it before (using Illumina's pre-calculated gtc_call_rate) did not handle 'zeroed-out SNPs'.
2) We fixed a bug that will change the value of some fields in the 'INFO' column of calculated VCFs.
3) We lowered call rate threshold used in autocall to determine if a gender call can be made in order to compensate for Illumina's GTC Call Rate not accounting for zeroed-out SNP

(These changes affect the outputs and truth will then need to be updated)

### CODE REVIEW

File: pipelines/broad/arrays/multi_sample/MultiSampleArrays.changelog.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.5.1
```
Comment (jessicaway) [inline pipelines/broad/arrays/multi_sample/MultiSampleArrays.changelog.md:1]: Is this a patch version if outputs are affected?

File: pipelines/broad/arrays/single_sample/Arrays.changelog.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+# 2.5.2
```
Comment (jessicaway) [inline pipelines/broad/arrays/single_sample/Arrays.changelog.md:1]: Same question

File: pipelines/broad/arrays/validate_chip/ValidateChip.changelog.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+# 1.14.1
```
Comment (jessicaway) [inline pipelines/broad/arrays/validate_chip/ValidateChip.changelog.md:1]: Here too

File: pipelines/broad/genotyping/illumina/IlluminaGenotypingArray.changelog.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+# 1.11.7
```
Comment (jessicaway) [inline pipelines/broad/genotyping/illumina/IlluminaGenotypingArray.changelog.md:1]: Here too

**Review by jessicaway — APPROVED** [review]
Looks good, just a question one question on the versioning for the affected pipelines

File: pipelines/broad/arrays/multi_sample/MultiSampleArrays.changelog.md (hunk 1 above)
Comment (gbggrant) [reply pipelines/broad/arrays/multi_sample/MultiSampleArrays.changelog.md:1]: Yeah, I thought about this.  It's very minor changes.  But changes nonetheless, so I'll bump the minor version.

**Review by nikellepetrillo — APPROVED** [review]
lgtm 👍

**Pull request merged**